docs/DeveloperGuide.md

```diff
@@ -621,13 +621,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ## **Appendix D: Non-Functional Requirements** <a name="appen-d"></a>
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 tasks/contacts/classes without a noticeable sluggishness in performance for
- typical usage.
-3.  Users of the app find the app intuitive and easy to use
-4.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should
+2.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should
```

Comment:
Think we can bold the `above average typing speed`

Reply:
oh yeah!